.pylintrc

```diff
@@ -178,6 +178,9 @@ disable=format,
         # we use it to speedup/optimize
         import-outside-toplevel,
         fixme,
+        # temporarily; should be fixed in pylint 2.6.1 or later
+        # (see https://github.com/PyCQA/pylint/issues/3882)
+        unsubscriptable-object,
```

Comment:
Don't think this is something that needs to be committed as it is not failing in CI?

Reply:
Reverted